packages/web/src/components/track/PremiumTrackSection.tsx

```diff
@@ -84,13 +84,17 @@ type PremiumTrackAccessSectionProps = {
   tippedUser: Nullable<User>
   goToCollection: () => void
   isOwner: boolean
+  className?: string
+  buttonClassName?: string
 }
 
 const LockedPremiumTrackSection = ({
   premiumConditions,
   followee,
   tippedUser,
-  goToCollection
+  goToCollection,
+  className,
```

Comment:
Nit: should be named `style` or something instead?

Reply:
And buttonStyle etc.

Reply:
on web we've mostly used `className` actually. there are 3 instance of `style` with web but we should do our best to be consistent. maybe we change those to `className` too